cfgmgr/intfmgr.cpp

```diff
@@ -196,11 +196,14 @@ bool IntfMgr::doIntfAddrTask(const vector<string>& keys,
             return false;
         }
 
-        // Set Interface IP except for lo
         if (!is_lo)
         {
             setIntfIp(alias, "add", ip_prefix.to_string(), ip_prefix.isV4());
         }
+        else
+        {
+            setIntfIp("lo", "add", ip_prefix.to_string(), ip_prefix.isV4());
+        }
```

Comment:
Why not just remove "if (!is_lo)" ?

Reply:
Initially I thought about that :-). But alias value is not 'lo'.
The 'alias' value for loopback interface is "Loopback0" as passed from the config command. The 'is_lo' boolean is set based by comparing the 'alias' name prefix with LOOPBACK_PREFIX (Loopback).
Hence the above change.